### PR TITLE
Http1 fallback

### LINKED ISSUE #59: Fetch providers via HTTP/1
This service connects to the providers via gRPC, we should try also connecting via HTTP/1 in case HTTP/2 is not available.

### PR DESCRIPTION
**Pull request by sekulicd**
This adds falling back mechanism to http1 if http2 is not available.

This closes #59 

@altafan please review.

### CODE REVIEW

File: pkg/tdex-market-loader/client.go [hunk 1]
```diff
@@ -92,6 +101,51 @@ func (t *tdexMarketLoaderService) FetchBalance(
 		},
 	})
 	if err != nil {
+		if s, ok := status.FromError(err); ok {
+			if s.Code() == codes.Unavailable {
+				requestData, err := json.Marshal(MarketReq{
```
Comment (altafan) [inline pkg/tdex-market-loader/client.go:106]: It's totally fine to use the types from the proto stubs `tdexv1`, we don't need these ad hoc files.

Comment (sekulicd) [reply pkg/tdex-market-loader/client.go]: Done

File: pkg/tdex-market-loader/types.go [hunk 2]
```diff
@@ -0,0 +1,101 @@
+package tdexmarketloader
```
[the remark this answers is not in the document]
Comment (sekulicd) [reply pkg/tdex-market-loader/types.go]: Done

**Comment by sekulicd** [comment]
Changes reverted as json responses cant be marshalled to proto types correctly cause of proto json tags.

**Comment by tiero** [comment]
I know is best to avoid extra boiler-plate when possible, But I am more in favor to have dedicated go types and utils to go from/to proto, instead of reusing it

**Review by altafan — APPROVED** [review]

**Pull request merged**